server/src/domain/facial-recognition/facial-recognition.service.spec.ts

```diff
@@ -169,6 +188,24 @@ describe(FacialRecognitionService.name, () => {
   });
 
   describe('handleRecognizeFaces', () => {
+    it('should return if machine learning is disabled', async () => {
+      const configCoreSpy = jest.spyOn(SystemConfigCore.prototype, 'getConfig').mockResolvedValue({
+        machineLearning: {
+          enabled: false,
+          clipEncodeEnabled: false,
+          facialRecognitionEnabled: false,
+          tagImageEnabled: false,
+          url: '',
+        },
+      } as SystemConfig);
```

Comment:
Can you mock the config repository instead? You can mock values in the load method?

Reply:
Yeah, that's much cleaner. Thanks!